mail-mta/notqmail/notqmail-9999.ebuild

```diff
@@ -0,0 +1,181 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+GENQMAIL_PV=20191010
+QMAIL_SPP_PV=0.42
+
+QMAIL_TLS_PV=20190517
+QMAIL_TLS_F=notqmail-1.07-tls-${QMAIL_TLS_PV}.patch
+
+QMAIL_BIGTODO_PV=103
+QMAIL_BIGTODO_F=big-todo.${QMAIL_BIGTODO_PV}.patch
+
+QMAIL_LARGE_DNS='qmail-103.patch'
```

Comment:
Is there a reason not using `" "` here? (only eyeable small nitpick I found)

Reply:
just copied from netqmail, will change that to ""